src/openzeppelin/utils.cairo

```diff
@@ -1,10 +1,50 @@
+use starknet::call_contract_syscall;
+use starknet::ContractAddress;
+use starknet::SyscallResult;
+use option::OptionTrait;
 use array::ArrayTrait;
 use array::SpanTrait;
 use box::BoxTrait;
-use option::OptionTrait;
 mod constants;
 mod serde;
 
+fn try_selector_with_fallback(
+    target: ContractAddress, snake_selector: felt252, camel_selector: felt252, args: Span<felt252>
+) -> SyscallResult<Span<felt252>> {
+    match call_contract_syscall(target, snake_selector, args) {
+        Result::Ok(r) => Result::Ok(r),
+        Result::Err(r) => {
+            if *r.at(0) == 'ENTRYPOINT_NOT_FOUND' {
+                return call_contract_syscall(target, camel_selector, args);
+            } else {
+                Result::Err(r)
+            }
```

Comment:
```suggestion
        Result::Ok(ret) => Result::Ok(ret),
        Result::Err(errors) => {
            if *errors.at(0) == 'ENTRYPOINT_NOT_FOUND' {
                return call_contract_syscall(target, camel_selector, args);
            } else {
                Result::Err(errors)
            }
```

I think this may be easier to read.